### ISSUE TITLE
UX: analyzer.log shown by default for basics/excercise2

### ISSUE THREAD

**Issue by awelzel** [zeek/try-zeek#64]
When running `basics/exercise2: Exercise`, the `analyzer.log` shows by default. That basically just shows the binpac exceptions observed from the PE analyzer. From a UX and marketing perspective that's pretty rough - that log isn't very interesting and even more likely shows entries about things that might not work quite right (observed violation/binpac exception, ...).

![Screenshot-20231116-115333](https://github.com/zeek/try-zeek/assets/2100509/e380b9ef-58ff-4897-8c1d-6bfb29395c69)

@JustinAzoff also mentioned it would make sense to just select `conn` by default. Could also order some popular logs first on the left ( `conn`, `ssl`, `dns`, `ssl`, `notice`). And then order some unpopular logs further to the right `dpd.log`, `analyzer.log`, `broker.log`, `cluster.log`, `config.log`, ...

**Comment by JustinAzoff** [comment]
I think what has happened is that `conn` used to always be first, just due to it being alphabetically before everything else, but now there's other stuff on by default.  Selecting conn by default if it exists shouldn't be tooo hard. the `BroFileViewer` component is the thing that renders that table.  Looking at the docs (at least for the latest version of react-bootstrap) adding this may be enough to do this:

```
      defaultActiveKey="conn"
```

Also, the run script has always done this:

```
#remove useless output files
rm -f loaded_scripts.log packet_filter.log notice_policy.log
```

but I don't know if removing analyzer.log is a good idea.

**Comment by JustinAzoff** [comment]
Should be fixed in 2adabdcd1e4c8b2f8f9d8f9674918a8d2bac01dc

**Comment by JustinAzoff** [comment]
Updated try.zeek.org.

Looks good.
